rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -19,6 +19,7 @@ def fixture_ethereumtokens(ethereum_inquirer, database, inquirer):  # pylint: di
     return EthereumTokens(database, ethereum_inquirer)
 
 
+@pytest.mark.vcr
```

Comment:
lol man no this can't work. This is detecting tokens which queries the last chain state which is always changing. This is not what VCR is for. This is not repeatable.

Reply:
Okay as discussed in the call let's go with this. Use vcr anywhere we can and avoid any complicated mocks.